ui/src/app/base/components/Login/Login.tsx

```diff
@@ -89,34 +90,37 @@ export const Login = (): JSX.Element => {
                   Go to login page
                 </Button>
               ) : (
-                <FormikForm<LoginValues>
-                  errors={error}
+                <Formik
```

Comment:
This PR is so huge mostly because of indentation changes. What's going on is that each form gets wrapped in `<Formik>` and the `Formik` props (`initialValues`, `onSubmit` etc.) get moved to `<Formik>`.